### DISCUSSION TITLE
Auto generated file [NameSpace].scp.css

### DISCUSSION THREAD

**Discussion by StefanJansson** [microsoft/fluentui-blazor#2290] [General]
In app.razor we have a default link to a stylesheet which is auto generated

`<link` rel="stylesheet" href="/[NameSpace].styles.css" />`

and the content looks like this

`@import '_content/Microsoft.FluentUI.AspNetCore.Components/Microsoft.FluentUI.AspNetCore.Components.bundle.scp.css';
@import '_content/[NameSpace]/css/_content/Microsoft.FluentUI.AspNetCore.Components/Microsoft.FluentUI.AspNetCore.Components.bundle.scp.css';`

Second row creates a 404

**Comment by vnbaaij** [comment]
That is not what I'm seeing in the demo site (for example). In 

In `FluentUI.Demo.Client.styles.css`
```
@import '_content/FluentUI.Demo.Shared/FluentUI.Demo.Shared.bundle.scp.css';
@import '_content/Microsoft.FluentUI.AspNetCore.Components/Microsoft.FluentUI.AspNetCore.Components.bundle.scp.css';
```

The `_content/[namespace]/css/_content...` is weird. No idea where that is coming form. but seems more like a solution/project issue than a library issue (I understand you put in [namespace] as a palceholder for the real name).

**Reply by StefanJansson** [reply]
Probably so, strange though. thx
